src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/PureSphericalHarmonic.cpp

```diff
@@ -0,0 +1,75 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/CurvedWaveEquation/PureSphericalHarmonic.hpp"
+
+#include <cstddef>
+#include <pup.h>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Evolution/Systems/CurvedScalarWave/Tags.hpp"
+#include "NumericalAlgorithms/Spectral/SwshInterpolation.hpp"
+#include "Utilities/MakeWithValue.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace CurvedScalarWave::AnalyticData {
+
+PureSphericalHarmonic::PureSphericalHarmonic(const double radius,
+                                             const double width,
+                                             std::pair<size_t, int> mode,
+                                             const Options::Context& context)
+    : radius_(radius), width_sq_(width * width), mode_{std::move(mode)} {
```

Comment:
Now clang-tidy complains. I think restore the `std::move`. The important thing was to use `mode_` in the body of the function to avoid the use-after-move (still don't understand why clang-tidy didn't complain about that before).

Reply:
it appears to be because `std::pair` simply isnt trivially copyable:

https://stackoverflow.com/questions/58283694/why-is-pair-of-const-trivially-copyable-but-pair-is-not

Reply:
The move is fine, but I wonder why clang-tidy didn't complain about the use-after-move.